components/form-control/script.js

```diff
@@ -58,6 +74,7 @@ class GamefaceFormControl extends HTMLElement {
         this.traverseFormElements(this, (element) => {
             //We dont need data from invalid elements and elements that are having type "submit"
             if (!this.isValidFromElement(element) || this.isElementUsedToSubmit(element)) return;
```

Comment:
```suggestion
            if (!this.isValidFormElement(element) || this.isElementUsedToSubmit(element)) return;
```
Please, change everywhere - From -> Form.

Reply:
Changed from from to form.